soteria-rust/scripts/parselog.py

```diff
@@ -417,6 +417,8 @@ def sort_items(item: tuple[str, str | tuple[TestInfo, TestInfo]]):
         if isinstance(info, str):
             return info
         only_before, only_after = info
+        if only_after.__len__() == 0:
```

Comment:
```suggestion
        if len(only_after) == 0:
```
not very pythonic :p